src/concatAll.js

```diff
@@ -0,0 +1,34 @@
+import { concat, identical, identity, pipe, when, reduce } from 'ramda';
+
+import stubNull from './stubNull';
+import fl from './fantasy-land/mapping';
+
+const nullSemigroup = { [fl.concat]: identity };
+
+/**
+ * Returns the result of concatenating all semigroups (such as arrays or strings) in passed foldable (such as an array).
+ * Returns null if empty foldable was passed.
+ *
+ * @func concatAll
+ * @memberOf RA
+ * @since {@link https://char0n.github.io/ramda-adjunct/2.6.0|v2.6.0}
+ * @category List
+ * @sig Semigroup s => Foldable s f => f -> (s | null)
+ * @param {Foldable<Semigroup>} foldable foldable with semigroups to concatenate
+ * @return {Semigroup|null} concatenated semigroups, or null, if empty foldable was passed
+ * @see {@link http://ramdajs.com/docs/#defaultTo|defaultTo}
```

Comment:
why defaultTo ?

Reply:
Because it's a way to get rid of the returned `null`.

Reply:
Yes but it's just an implementation detail. Consumer of the api should not be conserned with that. Actually I'm more inclined to use `undefined` instead of null. I've used some ramda functions today and noticed the ramda is never returning `null`. Always undefined. 

http://ramdajs.com/docs and search for `| undefined`

Reply:
Agree with `undefined`.

Well, actually the user of it should be aware of concerned that it'll return null or undefined when it receives empty foldable.

Reply:
Yes but link to `defaultTo` using `@see` doesn't do the job. Actually I think that there should only bee two `@see` tags with reference to `R.concat` and `RA.concatRight`. We should also add `@see` to `concatRight` that referece to `RA.concatAll`.

Reply:
I don't think it's necessary to reference `RA.concatRight`.

Reply:
Also, I'm of two minds about whether or not `@see` `R.unnest` and `R.join`, from one perspective, `concatAll` does a similar thing, except it has no sensible return value for empty foldable. From the other, `R.unnest` and `concatAll` have significantly different requirements about their parameters (`Chain` vs `Foldable Semigroup`).